vpc-terraform/ansible/playbook/nginx-role/tasks/main.yml

```
@@ -0,0 +1,15 @@
+---
+  - name: Install nginx 
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)